code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1202,6 +1202,58 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 
 	SSblackbox.add_details("admin_toggle","Toggled Hub Visibility|[GLOB.hub_visibility]") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+
+/client/proc/reset_atmos()
+	set name = "Clean Air"
+	set category = "Special Verbs"
+	set desc = "Cleans the air in a radius of harmful gasses like plasma and n2o "

```

Comment:
more a nitpick, but the string here probably shouldn't have trailing whitespace

Compare this:
```dm
set desc = "Cleans the air in a radius of harmful gasses like plasma and n2o "
```
to this:
```dm
set desc = "Cleans the air in a radius of harmful gasses like plasma and n2o"
```